assets/server/static/js/popup/main.js

```diff
@@ -37,6 +37,14 @@ document.addEventListener("DOMContentLoaded", async function () {
 
   // Call the function when select new category.
   categorySelect.addEventListener("change", updatePlaceholder);
+
+  const resetButton = document.querySelector('#resetButton');
+  resetButton.addEventListener("click", function () {
```

Comment:
it would be more appropriate to bind to the form reset event: https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset_event

Reply:
update in the latest version